.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Something is occurring that I think is wrong
+title: ''
+labels: "\U0001F41B bug"
+assignees: ''
+
+---
+
+## What is the current behavior?
+
+> What's happening that seems wrong?
+
+## Steps to reproduce
+
+> To make it faster to diagnose the root problem. Tell us how can we reproduce the bug.
+
+## Expected behavior
+
+> What would you expect to happen when following the steps above?
+
+## Please tell us about your environment
+
+> We want to make sure the problem isn't specific to your operating system or programming language.
+  
+- **Operating System/Version:** Windows 10
+- **Language:** [all | TypeScript | Python | PHP | etc]
+- **Browser:** Chrome
```

Comment:
```suggestion
> We want to make sure the problem isn't specific to your operating system.
  
- **Operating System/Version:** Windows 10
- **Browser:** Chrome
```
I'd also vote to change Chrome to Firefox Just Because (cc @bowlofeggs)

Reply:
Chrome is a pile of unmaintained vendored dependencies.